inventory-generation/tower-jobs-schedules/templates/tower-jobs-schedules.yaml.j2

```diff
@@ -0,0 +1,113 @@
+#jinja2: trim_blocks:False
+---
+start_date: "{{ (start_date | regex_replace('^(.*)T.*$', '\\1') | to_datetime('%Y-%m-%d')).strftime('%d %b %Y') }}"
+end_date: "{{ (end_date | regex_replace('^(.*)T.*$', '\\1') | to_datetime('%Y-%m-%d')).strftime('%d %b %Y') }}"
+archive_date: "{{ (archive_date | regex_replace('^(.*)T.*$', '\\1') | to_datetime('%Y-%m-%d')).strftime('%d %b %Y') }}"
+
+delete_missing_items: false
+
+ansible_tower:
+  url: "{{ ansible_tower_url }}"
+  admin_password: "{{ ansible_tower_admin_password }}"
+  projects:
+    - name: "infra-ansible"
+      description: "Infra Ansible"
+      scm_type: "git"
+      scm_url: "https://github.com/paulbarfuss/infra-ansible.git"
+      scm_branch: "add-tower-schedule-module"
+      organization: "{{ organization }}"
+    - name: "{{ customer_name }} {{ project_name }} Project"
+      description: "{{ description }}"
+      organization: "{{ organization }}"
+      scm_type: "git"
+      scm_url: "{{ url }}"
+      scm_branch: "master"
+      scm_credential_name: "{{ scm_credential_name }}"
+      scm_update_on_launch: true
+  job_templates:
+    - name: "Email and notify list of users - {{ customer_name }} {{ project_name }}"
+      description: "Job Template to email a list of users"
+      inventory: "{{ customer_name }} {{ project_name }} Inventory"
+      project: "infra-ansible"
+      playbook: "playbooks/notifications/email-notify-list-of-users.yml"
+      credential: ""
+      ask_variables_on_launch: true
+  inventories:
+    - name: "{{ customer_name }} {{ project_name }} [mail-host] Inventory"
+      variables: ""
+      organization: "{{ organization }}"
+      sources:
+        - name: "{{ customer_name }} {{ project_name }} [mail-host] Inventory Source"
+          description: "GitLab inventory source for the {{ customer_name }} {{ project_name }} engagement"
+          credential: "{{ scm_credential_name }}"
+          source: "scm"
+          source_project: "{{ customer_name }} {{ project_name }} Project"
+          source_path: "iac/inventories/mail-host/inventory/hosts"
+          update_on_launch: true
+          source_vars: |-
+            ---
+  schedules:
```

Comment:
As discussed, these schedules should be coming from a configuration repo rather than the automation implementation. Let's write a story to do a follow-up delivery on this.